.travis.yml

```diff
@@ -15,9 +15,9 @@ cache:
 matrix:
   include:
     - osx_image: xcode12
-      env: BUILD_SDK=iphonesimulator14.0 DESTINATION="OS=14.0,platform=iOS Simulator,name=iPhone 8" FULL_TESTS="true" MAIN_STAGE="true"
+      env: SWIFT=5.0 BUILD_SDK=iphonesimulator14.0 DESTINATION="OS=14.0,platform=iOS Simulator,name=iPhone 8" FULL_TESTS="true" MAIN_STAGE="true"
     - osx_image: xcode12
-      env: BUILD_SDK=iphonesimulator14.0 DESTINATION="OS=13.5,platform=iOS Simulator,name=iPhone 8" FULL_TESTS="false" MAIN_STAGE="false"
+      env: SWIFT=4.2 BUILD_SDK=iphonesimulator14.0 DESTINATION="OS=13.5,platform=iOS Simulator,name=iPhone 8" FULL_TESTS="false" MAIN_STAGE="false"
```

Comment:
This isn't enough, if you look at https://github.com/Skyscanner/SkyFloatingLabelTextField/blob/master/Rakefile you'll see I am also passing down the swift verison

Reply:
I guess that's the missing part to pass CI then